testing/util/service.py

```diff
@@ -93,8 +93,8 @@ def run(self):
         # TODO: Read database URI from application configuration file.
         #       Otherwise, pass "create_schema=True" through `start_process` or app config.
         #       However, this might be dangerous?
-        DatabaseAdapter(uri="sqlite:///secpi-database-testing.sqlite").connect().setup()
-
+        # DatabaseAdapter(uri="sqlite:///secpi-database-testing.sqlite").connect().setup()
+        DatabaseAdapter(uri="mysql+pymysql://secpi:secret@localhost/secpi_test").connect().setup()
```

Comment:
Nice spot. I should finally implement the `TODO` admonition, to reduce maintenance burdens on this end.
```suggestion
        DatabaseAdapter(uri="mysql+pymysql://secpi:secret@localhost/secpi-testdrive").connect().setup()
```